.gitignore

```diff
@@ -3,6 +3,7 @@
 # folders
 !/gradle
 !/src
+!/ide
```

Comment:
please don't open up the ide folder as a whole, but only the CodeStyle file... People like me will forget that at some point and accidently commit other IntelliJ Files.